packages/server/src/api/storage/sql/migrations/20191009-project-token.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
    */
   up: async (queryInterface, Sequelize) => {
     await queryInterface.addColumn('projects', 'token', {type: Sequelize.UUID()});
-    await queryInterface.bulkUpdate('projects', {token: Sequelize.col('id')}, {token: null});
+    await queryInterface.bulkUpdate(
+      'projects',
+      {token: Sequelize.col('id')},
+      {token: null},
+      {type: Sequelize.QueryTypes.BULKUPDATE}
```

Comment:
super weird that sequelize makes you do this when the name of the method is already bulkUpdate...

Reply:
Yes, i read for a while because without the type you get an error. 
I think it was fixed in Sequelize V5